lib/input.js

```diff
@@ -31,6 +31,13 @@ const _createInputDescriptor = function _createInputDescriptor (input, inputOpti
         throw new Error('Invalid density (1 to 2400) ' + inputOptions.density);
       }
     }
+    // page number
+    if (is.defined(inputOptions.page)) {
+      if (is.integer(inputOptions.page)) {
+        inputDescriptor.page = inputOptions.page;
+      }
+      // todo: check page range?
```

Comment:
Yes please, between 0 and [100000](https://github.com/jcupitt/libvips/blob/master/libvips/foreign/tiffload.c#L101).